### ISSUE TITLE
app-1.21.2 refuses administrator access (any and all)

### ISSUE THREAD

**Issue by isibley765** [atom/atom#16104]
### Description

Hey guys, so I'm on Windows 10 and wanted to try to get a formatter for jsx files on Atom so I could start using/learning Reactjs. I've already done it on my Ubuntu boot (dual boot computer), and from what I could tell running the installer again was supposed to update it(?). The installer downloads fine, but fails: 
`There was an error while installing the application. Check the setup log for more information and contact the author`
The log is long, so it's attached.
[SquirrelSetup.log](https://github.com/atom/atom/files/1443663/SquirrelSetup.log)

I wasn't looking to dive into this, and I didn't have it too customized, so  I next defaulted for deleting & reinstalling atom to see if that would work.
It didn't.
When I went to uninstall, I got an error message again:
`Windows cannot find 'C:\Users\<me>\AppData\Local\atom\Update.exe'. Make sure you typed the name correctly, and then try again.`

I think it may have been wiped during the installing trials. Actually, I think most of the files got wiped at some point, but that's besides the point/about to get there.
I then tried going into the file directory itself, and I think I found the main issue; besides there being nothing else in the atom file, there was a folder that I couldn't enter, delete, change properties on or anything. I'm given an `Access is denied` error any time I try anything, and all of the `.bat` 'run as administrator' hacks on Stack Overflow (deleting files or changing permissions) result in access errors.
When I try to open its security and look at who's allowed to look at it, I get `You do not have permission to view or edit this object's permissions settings.` even when I 'Continue' as an admin.

To make things even better, I've now deleted every file other than this, so I can't even use Atom anymore. I'm not sure what's going on, but Windows isn't having it, and my resources/family are all much more familiar with Linux permissions overriding than they are with Windows.

I'm not sure if there's anything you guys can really do at this point unless someone's a Window's whiz, but I figured I'd at least let you all be aware of it. It might have been another program rampaging through my system, or something might have gone awry inside of Atom's background workings or install processes, I couldn't tell you.

### Steps to Reproduce
I haven't the foggiest idea how the permissions got so messed up. I wish I could tell you.

**Expected behavior:** 
Folder deletes
**Actual behavior:** [What actually happens]
Folder remains
**Reproduces how often:** [What percentage of the time does it reproduce?]
100%
### Versions
1.21.2? That's the file name (Windows 10), but I can't see anymore (cause it's all deleted now...)

**Comment by rsese** [comment]
Thanks for the report @isibley765 - do you have any anti-virus software running?  If so, can you quit Atom, temporarily disable your anti-virus, and then try to install again?

**Comment by ashrafhasson** [comment]
SImilar issue face on Mac 10.11.6, had to disable anti-virus to have Atom installer upgrade from 1.21 to 1.22.0

**Comment by isibley765** [comment]
I probably do somewhere in the background, but I haven't run into it before in the past couple years I've had this computer.
Atom was also entirely uninstalled besides that one rogue folder, so I wouldn't have been able to even have it running to quit temporarily. But I get where you're going.

I lucked out though, I'm not sure if it was Windows cleaning up when it was powering down or Linux just not being prone to the same bug, but I logged onto my Ubuntu boot next. The Windows partition is visible to it, and on a whim I went to go look at that folder -- sure enough, it was gone.

I removed the atom folder completely from there and was able to install it next time I hopped onto my Windows boot.
